toolbox/gpu-operator/run_matrix_mult.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
+
+if [ "$#" -gt 1 ]; then
+    echo "FATAL: expected 0 or 1 parameter ... (got $#: '$@')"
+#   echo "Usage: $0 [gpu-burn runtime]"
+    echo "Usage: $0 [matrix-mult GPU]"
+    exit 1
+elif [ "$#" -eq 1 ]; then
+#   ANSIBLE_OPTS="${ANSIBLE_OPTS} -e gpu_burn_time=$1"
+    ANSIBLE_OPTS="${ANSIBLE_OPTS} -e matrix_mult_time=$1"
+    echo "Running GPU Matrix Mult for ${1} seconds."
+fi
+
+#exec ansible-playbook ${ANSIBLE_OPTS} playbooks/gpu-burn.yml
+exec ansible-playbook ${ANSIBLE_OPTS} playbooks/matrix-mult.yml
```

Comment:
the comments can be removed in this file,
and
```"Usage: $0 [matrix-mult GPU]"```
the argument can be `[matrix-mult runtime]`